streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -406,7 +403,12 @@ private static VerificationResult verifyAll(final Map<String, Set<Integer>> inpu
         boolean pass;
         try (final PrintStream resultStream = new PrintStream(byteArrayOutputStream)) {
             pass = verifyTAgg(resultStream, inputs, events.get("tagg"));
-            pass &= verifySuppressed(resultStream, "min-suppressed", inputs, events, SmokeTestDriver::getMin);
+            pass &= verifySuppressed(resultStream, "min-suppressed", events);
+            pass &= verify(resultStream, "min-suppressed", inputs, events, windowedKey -> {
+                final String unwindowedKey = windowedKey.substring(1, windowedKey.length() - 1).replaceAll("@.*", "");
+                return getMin(unwindowedKey);
+            });
```

Comment:
Decomponsed `verifySuppressed` and `verify`, so that we can just verify that `sws-suppressed` is actually suppressed, without verifying the results (which we would have to compute from first principles)